focus-ios/Blockzilla/Modules/WebView/LegacyWebViewController.swift

```diff
@@ -154,6 +154,18 @@ class LegacyWebViewController: UIViewController, LegacyWebController {
         configuration.allowsInlineMediaPlayback = true
         configuration.ignoresViewportScaleLimits = true
 
+        // For consistency we set our user agent similar to Firefox iOS.
+        //
+        // Important to note that this UA change only applies when the webview is created initially or
+        // when people hit the erase session button. The UA is not changed when you change the width of
+        // Focus on iPad, which means there could be some edge cases right now.
+
+        if UIDevice.current.userInterfaceIdiom == .pad {
+            configuration.applicationNameForUserAgent = "Version/15.0 Safari/605.1.15"
```

Comment:
Maybe a stupid question but: Why `Version/15.0`? It looks like you are imitating Safari's user agent, but it is already using `Version/18.4`. Firefox also already has `Version/16.0`. Doesn't this lead to the original problem repeating itself in the short term? Why does the number have to be smaller compared to Safari and Firefox?

Reply:
(Ideally it would use the version from the actual OS it's running, to e.g. exhibit Version/15.6 on iOS 15.8 and Version/18.3 on iOS 18.3.1 that are the correct Safari equivalents for the WKWebKit powering it, but guess that's easier said than done… I have a ticket in the works for that so I'll follow up…)

Here it's basically to match what's sent for other forms next to it, but yea both should be bumped in one go to match FxiOS:

```suggestion
            configuration.applicationNameForUserAgent = "Version/16.0 Safari/605.1.15"
```